File: android/app/src/main/kotlin/com/example/want/WebViewActivity.java
package com.example.want;

import android.app.Activity;
import android.os.Bundle;
import android.webkit.WebView;
import android.webkit.WebViewClient;


import java.util.HashMap;

public class WebViewActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_splash);
        WebView webview = findViewById(R.id.webview);

        Bundle extras = getIntent().getExtras();
        HashMap<String,String> arguments = (HashMap<String, String>) extras.getSerializable("arguments");

        webview.setWebViewClient(new WebViewClient());//添加WebViewClient实例
        webview.loadUrl(arguments.get("url"));//添加浏览器地址
    }
}

File: android/app/src/main/kotlin/com/example/want/WebViewActivity.kt
package com.example.want

import android.app.Activity
import android.os.Bundle
import android.webkit.WebChromeClient
import android.webkit.WebView
import com.example.untitled.R
import java.util.HashMap

class WebViewActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.webview)
        val webView = findViewById<WebView>(R.id.webView)
        val intent = intent
        val map = intent.getSerializableExtra(pluginParams)

        var hashMap = HashMap<String, String>()
        if (map != null) {
            @Suppress("UNCHECKED_CAST")
            hashMap = map as HashMap<String, String>
        }
        val webViewClient = WebChromeClient()
        webView.webChromeClient = webViewClient
        webView.loadUrl(hashMap["url"].toString())
    }
}